CMSgov/onemac
# OY2-10086 Fix the Attachment bucket functionality

Story: https://qmacbis.atlassian.net/browse/OY2-10086
Endpoint: https://d1l9gqzzztogjx.cloudfront.net/

### Details
Update the Application so that it uses the environment variable for the correct S3 bucket information when sharing links to the attachments.

### Changes
- null the returned attachment URL from the backend
- use the bucket variable and the submitting user id to create the correct link.

### Implementation Notes
- This update is made as a fix for the nationwide release as part of the attachment migration.

### Test Plan
1. Login as statesubmitteractive@cms.hhs.local and successfully submit something with attachments.
2. Login as helpdeskactive@cms.hhs.local, view that submission
3. Click on the attachments and verify that you can view/download them.
